Runtime/Cesium3DTile.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using Reinterop;
+using System;
+using Unity.Mathematics;
+using UnityEngine;
+
+namespace CesiumForUnity
+{
+    [ReinteropNativeImplementation("CesiumForUnityNative::Cesium3DTileImpl", "Cesium3DTileImpl.h", staticOnly: true)]
+    public partial class Cesium3DTile
```

Comment:
If this class is public, can we write some documentation ? It seems to be a wrapper around a native Cesium3DTile, so people should know not to create one randomly, for example.

I also initially thought this would have a one-to-one correspondence with a 3D tile (so as many C# tiles as C++ tiles), so I was very confused when the Unity adaptor kept changing the pointer of it. It's definitely better to create just one class and reuse it, but the naming is a bit deceptive. Maybe it can be renamed to something like `Cesium3DTileHandler` or `Cesium3DTileView`? Or if we keep the name, it should at least be extremely well documented.

Reply:
I think it's reasonable to call it a `Cesium3DTile` even though it doesn't have a 1:1 correspondence with the C++ tiles. In most cases, users shouldn't care. But I updated the doc to make it a little clearer, and also iimproved the ShouldExclude method doc to hopefully make it clearer that you can't hold on to the tile for later.